dizoo/league_demo/league_demo_ppo_config.py

```diff
@@ -1,12 +1,13 @@
 from easydict import EasyDict
+from torch.nn.modules.activation import Threshold
 
 league_demo_ppo_config = dict(
     exp_name="league_demo_ppo",
     env=dict(
         collector_env_num=8,
         evaluator_env_num=10,
         n_evaluator_episode=100,
-        stop_value=[-0.01, -5],
+        stop_value=[-10.1, -5.1],
```

Comment:
Why did you change the first stop value?

Reply:
Because the eval policy is changed so that the expected reward in Nash Equilibrium should be changed.

Reply:
Sounds good. Please make sure it can converge.

Reply:
The calculation formula is as follows. 
![e54d5f489daddff60fddb1b1627a79ea](https://user-images.githubusercontent.com/53364123/131822406-0d110ba4-a0e1-4399-b6a5-43ad00bce87d.jpg)